Automattic/jetpack
# Add form responses v2 endpoint

## Proposed changes:
This PR adds wpcom/v2/form-responses endpoint as the target for WPCOM's D102076-code 

### Other information:


## Does this pull request change what data or activity we track or use?


## Testing instructions:

On docker, add these 2 lines on `plugins/jetpack/modules/contact-form.php`:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

Then head to http://localhost/wp-admin/admin.php?page=jetpack-forms to get a mock of the new inbox page. Open devtools and type this on the console:

```
wp.apiFetch({path: '/wpcom/v2/forms/responses'}).then(r=>console.log(r))
```

You should get a list of form responses.
